### PR TITLE
Add docs for pugWrapAttributesThreshold to README

### PR DESCRIPTION
**Pull request by lehni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,18 +152,52 @@ These additional options are specific to pug templates and can be configured in
...
+    input(type="text")
+    input(
+      type="text",
+      value="my_value",
+      name="my_name"
+    )
+    ```
+  - `2 … Infinity` → Same as above, just with different thresholds.  
 
 The definitions for these options can be found in [src/options/index.ts](https://github.com/prettier/plugin-pug/blob/master/src/options/index.ts)
```
Comment (Shinigami92) [inline README.md:202]: Could you also change this? `src/options/index.ts` is outdated :thinking: 
Maybe `src/options/*`? Or leave it completely?!

File: README.md [hunk 2]
```diff
@@ -109,11 +109,11 @@ These additional options are specific to pug templates and can be configured in
 
   Choices:
 
-  - `'always'` _default_ -> Always separate attributes with commas.  
+  - `'always'` _default_ → Always separate attributes with commas.
```
Comment (Shinigami92) [inline README.md:112]: I'm a little bit afraid of these arrows ^^'
Do thy work on any OS and always will be rendered correctly?

File: README.md [hunk 3]
```diff
@@ -152,18 +152,52 @@ These additional options are specific to pug templates and can be configured in
...
+    Example:
+    ```pug
+    input(type="text")
+    input(
+      type="text",
+      value="my_value",
+      name="my_name"
+    )
+    ```
+  - `2 … Infinity` → Same as above, just with different thresholds.
```
Comment (Shinigami92) [inline README.md:200]: Same for this `…`
Also IMO ranges are written with only two dots 🤣 `2 .. Infinity`

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (lehni) [reply README.md:112]: Good point. I think they're fine in 2020, but happy to switch back :)

File: README.md (hunk 3 above)
Comment (lehni) [reply README.md:200]: right! I'll switch

File: README.md (hunk 1 above)
Comment (Shinigami92) [reply README.md:202]: ok, leave it out

File: README.md (hunk 3 above)
Comment (lehni) [reply README.md:200]: But that's math notation then, and would need `[2 .. Infinity]`, looking like an array ;)

Comment (Shinigami92) [reply README.md:200]: 😏

File: README.md (hunk 1 above)
Comment (lehni) [reply README.md]: Done!

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**